### PR TITLE
 finished BossEvent packet

### PR DESCRIPTION
**Pull request by kroer**
I did not touch the BossBar class, but something was wrong with it.

### CODE REVIEW

File: src/MiNET/MiNET/Net/MCPE Protocol.cs [hunk 1]
```diff
@@ -6187,9 +6187,27 @@ protected override void ResetPacket()
...
+			UpdateProperties = 6,
+			UpdateStyle = 7,
+		}
 
 		public long bossEntityId; // = null;
-		public uint eventType; // = null;
+		public BossEvent eventType = BossEvent.Add;
+		public string bossName = "";
+		public float bossHealthPercent = 1.0f; // from 0 to 1.0
+		public ushort darkenScreen = 12; // forgot what is it
```
Comment (dktapps) [inline src/MiNET/MiNET/Net/MCPE Protocol.cs]: I'd be prepared to bet this is the same as PC, since it's an almost perfect copy pasta.

Comment (vchibikov) [reply src/MiNET/MiNET/Net/MCPE Protocol.cs]: Thanks. Check it today.

Comment (kroer) [reply src/MiNET/MiNET/Net/MCPE Protocol.cs]: Thanks. Check it out today.

Comment (kroer) [reply src/MiNET/MiNET/Net/MCPE Protocol.cs]: @dktapps I think you are right. But it changes nothing on the client side when I change it on the server.

Comment (dktapps) [reply src/MiNET/MiNET/Net/MCPE Protocol.cs]: I checked it myself and there are some minor differences, but you're right about it being useless.

File: src/MiNET/MiNET/Net/MCPE Protocol.cs [hunk 2]
```diff
@@ -6187,9 +6187,27 @@ protected override void ResetPacket()
 
 	public partial class McpeBossEvent : Packet<McpeBossEvent>
 	{
+		public enum BossEvent : uint
```
Comment (NiclasOlofsson) [inline src/MiNET/MiNET/Net/MCPE Protocol.cs]: You can't edit this file directly. It needs to be editied via XML.

**Comment by NiclasOlofsson** [comment]
Sorry, but I can't use this PR. Needs to be updated to current master and reworked from that. Also the protocol needs to be generated via the XML file.

**Pull request closed without merging**